packages/libs/client-utils/src/nodejs/read-file.ts

```diff
@@ -0,0 +1,13 @@
+import { readFileSync } from 'fs';
+
+interface IReadFileOptions {
+  encoding?: BufferEncoding | null;
+  flag?: string;
+}
+
+export const readFile = (
```

Comment:
this is redundant. only move yaml-convertor.ts to the nodejs folder and then export it via the index.ts.